pages/common/vagrant.md

```diff
@@ -0,0 +1,19 @@
+# vagrant
+
+> Configure lightweight, reproducible, and portable development environments
+
+- Create Vagrantfile in current folder with the given Vagrant box
+
+`vagrant init hashicorp/precise32`
```

Comment:
It is mentioned in all tutorials
https://docs.vagrantup.com/v2/getting-started/index.html

Frankly, it is just clean Ubuntu 12.04 :)